_partials/ubuntu_vscode.md

```diff
@@ -0,0 +1,83 @@
+
+## Visual Studio Code - Your text editor
+
+A text editor is one of the most important tools of a developer.
+Follow these instructions in the Terminal:
+
+```bash
+sudo snap install --classic code
+```
+
+:point_up: This command will ask for your password with: `[sudo] password for <username>:`. Don't panick! Calmly type your password key by key. You won't have a visual feedback (like little `*`), that's **perfectly normal**, keep on typing. When you're done, hit `Enter` :muscle:.
+
+In case you don't have `snap` installed - ask a TA for help.
```

Comment:
I think we should privilege `.deb` install over `snap` as it's the default recommended installation process. 

We can tell students to go to [VSCode download page](https://code.visualstudio.com/download), download the `.deb` file and execute it, just like the [macOS setup](https://github.com/lewagon/data-setup/pull/63/files#diff-eb1821ae8217ca53139a9b44d09a273852585b3001b5da68563825809c3bf76eR3-R7).

Reply:
Or what about everything executed from command line?

```bash
wget -q https://packages.microsoft.com/keys/microsoft.asc -O- | sudo apt-key add -
sudo add-apt-repository "deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main"
sudo apt update
sudo apt install code
```

Reply:
Yes, it's better. Let's go for it!